### PR TITLE
adding form index.html and form  style in style.css

### PR DESCRIPTION
**Pull request by Muniralsharif**

### CODE REVIEW

**Review by someyaaltous — APPROVED** [review]

**Review by DuaaH — APPROVED** [review]

File: public/style.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  }
+  
+  form.search::after {
+    content: "";
+    clear: both;
+    display: table;
+  }
+
+  .search{
+
```
Comment (jema28) [inline public/style.css:8]: Please delete all this free space above and below these two lines.

File: public/index.html [hunk 2]
```diff
@@ -3,10 +3,15 @@
...
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="style.css">
     <title>Football Teams</title>
   </head>
   <body>
-    <input type="text" name="countryname" />
-    <button type="button" class="search">search</button>
+    <form class="search" action="/" style="">
+      <input type="text" id="search" placeholder="Search.." name="search2">
```
Comment (jema28) [inline public/index.html]: Why is this called 'search2'?

File: public/style.css [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    padding: 10px;
+    font-size: 17px;
+    border: 1px solid grey;
+    float: left;
+    width: 80%;
+    background: #f1f1f1;
+  }
+  
+  form.search button {
+    float: left;
```
Comment (jema28) [inline public/style.css]: Why are you using float? Try using float or grid.

**Review by jema28 — CHANGES_REQUESTED** [review]

File: public/style.css (hunk 1 above)
Comment (Muniralsharif) [reply public/style.css:8]: @jema28  fixed thanks

File: public/index.html (hunk 2 above)
Comment (Muniralsharif) [reply public/index.html]: @jema28  I changed the name to be searchBox

File: public/style.css (hunk 3 above)
Comment (Muniralsharif) [reply public/style.css]: @jema28  no need for float, I remove it

File: public/style.css [hunk 4]
```diff
@@ -14,9 +14,13 @@ form.search input[type="text"] {
   float: left;
   width: 80%;
   background: #f1f1f1;
+  border-radius: 10px;
+
 }
 
 form.search button {
+    border-radius: 10px;
```
Comment (jema28) [inline public/style.css]: There's still a huge space here and problem with alignment.

**Review by jema28 — CHANGES_REQUESTED** [review]

File: public/style.css (hunk 4 above)
Comment (Muniralsharif) [reply public/style.css]: @jema28 fixed

**Review by jema28 — APPROVED** [review]

**Review by jema28 — CHANGES_REQUESTED** [review]
There's still a problem here: 'assest' folder still exists (so there are now two assets folders) and there's a space in css, line 42.

**Review by jema28 — APPROVED** [review]

**Pull request merged**